### PR TITLE
show inspection date in local timezone and inspector on submissions table

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

1 - show 'Inspection Date' in local timezone on Inspection History table.
The inspection date is stored as a UTC timestamp in the db.
On the pop up history table, it should be converted back to local timezone (ie -7 or -8 hrs)

2 - show assigned inspector on admin>submissions table

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/components/admin/SubmissionsTable.vue [hunk 1]
```diff
@@ -107,13 +108,16 @@ export default {
           const data = response.data;
           const submissions = Object.keys(data).map(k => {
             let submission = data[k];
+            let inspectorName = (submission.inspectionStatuses[0].inspectorName !== null) ? submission.inspectionStatuses[0].inspectorName : ' - ';
```
Comment (loneil) [inline app/frontend/src/components/admin/SubmissionsTable.vue:111]: Can just be a falsey check 
if(submission.inspectionStatuses[0].inspectorName)

Comment (TimCsaky) [reply app/frontend/src/components/admin/SubmissionsTable.vue:111]: oh yeah. i appreciate the reminder. thanks

Comment (loneil) [reply app/frontend/src/components/admin/SubmissionsTable.vue]: Could get undesired results with just the null check. Reminding myself what each condition evaluates to with that....

image

Comment (TimCsaky) [reply app/frontend/src/components/admin/SubmissionsTable.vue]: huh. ok. i wasnt aware of some of that. i'm definitely saving this image.

File: app/frontend/src/components/admin/SubmissionsTable.vue [hunk 2]
```diff
@@ -107,13 +108,16 @@ export default {
           const data = response.data;
           const submissions = Object.keys(data).map(k => {
             let submission = data[k];
+            let inspectorName = submission.inspectionStatuses[0].inspectorName ? submission.inspectionStatuses[0].inspectorName : ' - ';
```
Comment (jujaga) [inline app/frontend/src/components/admin/SubmissionsTable.vue:111]: This should be a const

**Pull request merged**